notebooks/official/matching_engine/intro-swivel.ipynb

```diff
@@ -153,7 +153,7 @@
       "source": [
```

Comment:
<p>network_name: my_network_name -&gt; network_name: e.g., my_network_name </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/193/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
done

Reply:
<p>close.</p><p>network_name: network_name: e.g -&gt; network_name: e.g.,</p><p> </p><p>Note you repeated network_name twice.</p>

Reply:
done